src/Bridge/EasyAdmin/src/Controller/MediaAdminController.php

```diff
@@ -298,6 +300,18 @@ public function list(AdminContext $adminContext, Request $request, string $key =
             default => 'explore',
         };
 
+        $page = max(1, $request->query->getInt('page', 1));
+        $perPage = $this->config->getPerPage() ?? 50;
```

Comment:
can you please set it to `20`, as this is the default easyadmin pagination size?